React/wbooks/src/Home/layout.js

```diff
@@ -16,7 +16,7 @@ function Home({ filterOption, searchText, handleFilterOptionChange, handleSearch
         onFilterOptionChange={handleFilterOptionChange}
       />
       <BookSearch searchText={searchText} onSearchTextChange={handleSearchTextChange} />
-      <BookContainer books={BOOKS} searchText={searchText} filterOption={filterOption} />
+      <BookContainer books={books} searchText={searchText} filterOption={filterOption} />
```

Comment:
You could do the book fetching in BookContainer instead of Home. That's the component that's going to use it. That way yo avoid passing books from component to component, only the one that really uses it should fetch it and read it from the store.